### DISCUSSION TITLE
Website not rendering properly

### DISCUSSION THREAD

**Discussion by jbkjr** [academicpages/academicpages.github.io#2157] [Q&A]
For some reason, my website (jbkjr.me, hosted through GH pages) is not rendering properly. It looks like this right now:

<img width="1264" alt="Screenshot 2024-05-29 at 1 44 50 PM" src="https://github.com/academicpages/academicpages.github.io/assets/33947792/bb51790c-5a9d-44dc-9dfa-46e78f2638c9">

The one clue I've been able to find is that if I turn off "enforce https" in my GH pages settings and visit http://jbkjr.me, it renders correctly, but not at https://jbkjr.me. So, it has something to do with https, but I don't know how to fix it. Any help on this is greatly appreciated!

**Comment by arceon09** [comment]
Hello! Wanted to know how you fixed the problem. I've been having the same issue.
Not only does it not show up properly, but the other pages do not load as well. 
https://arceon09.github.io/arceon09/

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/cba1ca81-068e-4c68-a5a3-51fb889c2a95">

**Reply by arceon09** [reply]
Fixed. Had something to do with the naming of the repo.

**Reply by evertbosdriesz** [reply]
I am having the same issue. Could you share how you fixed it in the end?

**Reply by rjzupkoii** [reply]
@evertbosdriesz Make sure you are following are the directions [from GitHub for a custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site) and the `_config.yml` file has the correct URL as well.
